trade-finance-manager-ui/server/controllers/case/activity/helpers/map-activities.ts

```diff
@@ -0,0 +1,43 @@
+import { ACTIVITY_TYPES, TfmActivity } from '@ukef/dtfs2-common';
+import { fromUnixTime } from 'date-fns';
+
+export const mapActivities = (activities: TfmActivity[] | undefined) => {
```

Comment:
Documentation? Not clear why this mapping exists and without going into the mapping and switch/case, what it does